### ISSUE TITLE
Enhancement: Support for setCustomValidity()

### ISSUE THREAD

**Issue by AzzaAzza69** [WebCoder49/code-input#40]
I currently use setCustomValidity against a textarea where I key in the string containing a JSON object.
eg
{ "prop": "value" }
Changing the textarea to a code-input gives J/S errors indicating the function is missing! 
Is it something you would consider implementing?
Would it also be easy enough to take attributes from the code-input and apply them to the interior textarea like form, required, maxlength, minlength, cols, rows, wrap, etc.

**Comment by WebCoder49** [comment]
Thank you for this suggestion. Seamless swapping of textareas for code-input elements is one of the main aims of the project, so I will make it a priority to add this.

However, I am afraid that I will be very busy until the end of June, so would appreciate it if you could add this via a pull request (you can override setCustomValidity in the `codeInput.CodeInput` class and follow the existing `name` attribute setup for the HTML attributes). Please let me know if you can't and I will work on it when I have time.

**Comment by WebCoder49** [comment]
To clarify this, these methods are being called on a normal code-input element:
```html
<code-input lang="JSON" id="input-json"    name="json"  placeholder="JSON File Contents"  required></code-input>
```

**Comment by AzzaAzza69** [comment]
sorry if I didn't explain correctly...
I was thinking just to take the attributes minlength, maxlength, required and copying them from the code-input (or the original textarea when you get the seamless swap done) to the textarea, it would respond.
Also if I used codeinput.setCustomValidity('message'), it would push the message onto the textarea.

**Comment by WebCoder49** [comment]
I have added this to the branch mentioned above. I will carry out some final testing then merge and release it when I have time.

**Comment by WebCoder49** [comment]
Thank you again for this suggestion; I have released it in [v1.5.0](https://GitHub.com/WebCoder49/code-input/releases). You can see a demo of how to use it [here](https://codepen.io/WebCoder49/details/JjmqjZv).
